packages/lib/slugify.ts

```diff
@@ -2,6 +2,9 @@
 // For eg:- "test-slug" is the slug user wants to set but while typing "test-" would get replace to "test" becauser of replace(/-+$/, "")
 
 export const slugify = (str: string, forDisplayingInput?: boolean) => {
+  if (!str) {
+    return "";
```

Comment:
This feels like it could result in letting bad data through. i.e. Instead of failing, this returns empty and stores empty slugs or something alike. Thoughts?

Reply:
Think this code was outside this PR - i just moved it